src/peft/tuners/tuners_utils.py

```diff
@@ -299,6 +299,16 @@ def unmerge_adapter(self):
             if isinstance(module, BaseTunerLayer):
                 module.unmerge()
 
+    def _unloading_checks(self, adapter_names: List[str]):
+        adapters_to_consider = adapter_names or self.active_adapters
+        is_modules_to_save_available = any(
+            self.peft_config[adapter].modules_to_save is not None for adapter in adapters_to_consider
```

Comment:
```suggestion
            self.peft_config[adapter].modules_to_save for adapter in adapters_to_consider
```

How about this change? This way, if `modules_to_save = []`, it is not counted here.